src/main/bvmui/package.json

```diff
@@ -11,7 +11,8 @@
     "lodash": "4.17.15",
     "vue": "2.6.10",
     "vue-router": "^3.1.3",
-    "vuex": "3.1.1"
+    "vuex": "3.1.1",
+    "yarn": "^1.21.1"
```

Comment:
Do we really need yarn?

Reply:
it was added by yarn automatically, reverted